config.md

```diff
@@ -127,4 +127,77 @@ For Linux-based systems the user structure has the following fields:
 Interpretation of the platform section of the JSON file is used to find which platform-specific sections may be available in the document.
 For example, if `os` is set to `linux`, then a JSON object conforming to the [Linux-specific schema](config-linux.md) SHOULD be found at the key `linux` in the `config.json`.
 
+## Configuration Schema Example
+
+Here is a full exmample `config.json` contents for reference (generated with `runc spec`).
+
+```json
+{
+	"version": "0.2.0",
+	"platform": {
+		"os": "linux",
+		"arch": "amd64"
+	},
+	"process": {
+		"terminal": true,
+		"user": {
+			"uid": 0,
+			"gid": 0,
+			"additionalGids": null
```

Comment:
If we're looking for a full config (i.e. excercising all available options), we should probably include a real `additionalGids` value (and similar for other optional fields).  If the intention is just to show an example that explores most of the available options for anchoring the individual snippets (like [this one][1]), then I'd probably just drop the no-op `additionalGids` (because the surrounding entries provide enough context to anchor the existing user snippet).

[1]: https://github.com/opencontainers/specs/blob/e79365a749f86254dec96609caff2a8acd15f689/config.md#process-configuration